library/src/main/java/com/chuckerteam/chucker/internal/ui/MainViewModel.kt

```diff
@@ -35,6 +41,16 @@ internal class MainViewModel : ViewModel() {
     val throwables: LiveData<List<RecordedThrowableTuple>> = RepositoryProvider.throwable()
         .getSortedThrowablesTuples()
 
+    suspend fun getAllTransactions(): List<HttpTransaction>? = withContext(Dispatchers.Default) {
```

Comment:
There is no need in specifying `Dispatcher`, since Room handles switching from UI thread for DAOs.
You can find more info here: https://medium.com/androiddevelopers/room-coroutines-422b786dc4c5

Reply:
thanks for the pointer, updated.